Podfile.lock

```diff
@@ -2,53 +2,53 @@ PODS:
   - CreatubblesAPIClient (0.2.0):
     - KeychainAccess (= 3.0.1)
     - ObjectMapper (= 2.0)
-    - RealmSwift (= 2.0.3)
+    - RealmSwift (= 2.10.1)
     - XCGLogger (= 4.0.0)
   - KeychainAccess (3.0.1)
-  - Nimble (5.0.0)
+  - Nimble (7.0.1)
   - ObjectMapper (2.0.0)
-  - Quick (0.10.0)
-  - Realm (2.0.3):
-    - Realm/Headers (= 2.0.3)
-  - Realm/Headers (2.0.3)
-  - RealmSwift (2.0.3):
-    - Realm (= 2.0.3)
-  - SwiftLint (0.19.0)
+  - Quick (1.1.0)
+  - Realm (2.10.1):
+    - Realm/Headers (= 2.10.1)
+  - Realm/Headers (2.10.1)
+  - RealmSwift (2.10.1):
+    - Realm (= 2.10.1)
+  - SwiftLint (0.22.0)
   - XCGLogger (4.0.0):
     - XCGLogger/Core (= 4.0.0)
   - XCGLogger/Core (4.0.0)
 
 DEPENDENCIES:
-  - CreatubblesAPIClient (from `https://github.com/creatubbles/ctb-api-swift.git`, branch `develop`)
+  - CreatubblesAPIClient (from `https://github.com/creatubbles/ctb-api-swift.git`, branch `feature/469_xcode_9`)
```

Comment:
Do we want to use that branch instead of develop? I mean, we will need to remember to change it in the future, right?

Reply:
I'll have to update it after merge (`Podfile.lock` is automatically generated file)